src/node_options.cc

```diff
@@ -410,11 +410,6 @@ EnvironmentOptionsParser::EnvironmentOptionsParser() {
             &EnvironmentOptions::experimental_global_navigator,
             kAllowedInEnvvar,
             true);
-  AddOption("--experimental-global-webcrypto",
-            "expose experimental Web Crypto API on the global scope",
-            &EnvironmentOptions::experimental_global_web_crypto,
-            kAllowedInEnvvar,
-            true);
```

Comment:
Can you keep the flag as a no-op please?

Reply:
I can, but would you please elaborate why?

Reply:
If the flag remains but is no longer functional users will not be notified of it when they upgrade.

Reply:
Ah okay, if the negative is still rejected with a noop, 👌